MREUnityRuntime/MREUnityRuntimeLib/Constants.cs

```diff
@@ -23,15 +23,26 @@ static Constants()
             SerializerSettings.Converters.Add(new PayloadConverter());
         }
 
+        /*
+         * Headers passed in MRE connect requests.
+         */
         internal const string SessionHeader = "x-ms-mixed-reality-extension-sessionid";
         internal const string PlatformHeader = "x-ms-mixed-reality-extension-platformid";
-        internal const string ProtocolVersionHeader = "x-ms-mixed-reality-extension-protocol-version";
```

Comment:
>ProtocolVersionHeader [](start = 30, length = 21)

We can't stop sending these without breaking old SDK versions. We'll have to make sure to message the community about this. Alternatively, can we keep sending protocol versions too for a couple weeks? That would prevent random breakage.